### PR TITLE
daniel_escobar_Session01

### PR DESCRIPTION
**Pull request by danielnamae**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# k8s-training
-Kubernetes Training exercises 
+Exercise 01
```
Comment (juan131) [inline README.md]: I'd be nice if you could info about how to use/deploy your K8s resources

File: ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: wordpress-ingress
+  namespace: exercise-01
+  labels:
+    app: wordpress
+  annotations:
+    nginx.ingress.kubernetes.io/affinity: cookie
```
Comment (juan131) [inline ingress.yaml]: Does it make sense to have this annotation without the complement annotations?

```
nginx.ingress.kubernetes.io/session-cookie-name
nginx.ingress.kubernetes.io/session-cookie-hash
```

File: wordpress-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    metadata:
+      labels:
+        app: wordpress
+        type: frontend
+
+    spec:
+      containers:
+      - image: bitnami/wordpress:latest        
+        name: wordpress
+        env:
```
Comment (juan131) [inline wordpress-deployment.yaml]: I'm missing some env. variables such as **WORDPRESS_USERNAME**, **WORDPRESS_PASSWORD**, **WORDPRESS_FIRST_NAME**, etc. so you can customise your WordPress site

File: wordpress-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                name: wordpress
+                key: blogname         
+
+          - name: WORDPRESS_TABLE_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: wordpress
+                key: prefixtable
+
+        ports:
```
Comment (juan131) [inline wordpress-deployment.yaml]: You can also expose the 443 port

**Comment by danielnamae** [comment]
I'm sorry, I will fit it when I arrive home. I was busy with current exercises and I forgot to change current structure.

**Comment by danielnamae** [comment]
I'm going to send the rigth files structure.

**Pull request closed without merging**